x-pack/plugins/security_solution/server/lib/timeline/saved_object_mappings/migrations/notes.test.ts

```diff
@@ -1,40 +0,0 @@
-/*
```

Comment:
I moved these tests to the utils file because the migration function was moved to the utils file as well.